src/OrbitGl/ThreadTrack.cpp

```diff
@@ -413,6 +413,14 @@ float ThreadTrack::GetHeaderHeight() const {
   return header_height;
 }
 
+float ThreadTrack::GetYFromDepth(uint32_t depth) const {
```

Comment:
I'm not sure I understand why this override is needed now. Isn't one of the reasons `GetHeaderHeight` was introduced to avoid overriding `GetYFromDepth` for every class deriving from `TimerTrack` and provide a single template method to customize the timer positioning?

This looks like `gap_between_tracks_and_timers` should be part of the header height calculation, so the logic to check for thread bars is only required in `GetHeaderHeight`.

Reply:
Yeah that thing is a bit funcy, but for now, we need to keep it as it is. It is all basically because of the last space between the event bar and the timers pane.
We only want to draw it if there are timers (otherwise, "empty" tracks would have some strange space). However, in the GetHeaderHeight, we don't know about whether we have a timers pane or not.

Reply:
Ok, SGTM, let's fix it the proper way in a later PR.